sdk/Sdk/Extensions/StringExtensions.cs

```diff
@@ -0,0 +1,52 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+namespace Microsoft.Azure.Functions.Worker.Sdk
+{
+    internal static class StringExtensions
+    {
+        /// <summary>
+        /// Returns a copy of the string in camelCase.
+        /// </summary>
+        public static string ToCamelCase(this string value)
```

Comment:
I was looking at how the in-proc SDK does this.... and all it does is lowercase the first letter. I think we should just copy that behavior so that they behave the same.

https://github.com/Azure/azure-functions-vs-build-sdk/blob/v3.x/src/Microsoft.NET.Sdk.Functions.Generator/AttributeExtensions.cs#L32

Reply:
Thanks Brett. Switched to that implementation.